modules/runner/src/main/java/org/apache/ignite/configuration/ConfigurationModule.java

```diff
@@ -23,7 +23,7 @@
  * Module is responsible for preparing configuration when module is started.
  *
  * Preparing configuration includes reading it from configuration file, parsing it and initializing
- * {@link Configurator} object.
+ * {@code Configurator} object.
```

Comment:
I can't find this class anywhere, are you sure this javadoc is up to date?

Reply:
No, it's not, I'll fix it, thank you!